_posts/2024-04-04-release-notes.md

```diff
@@ -0,0 +1,231 @@
+* `USN-6718-1` curl vulnerabilities:
+  * `CVE-2024-2004`: Usage of disabled protocol
+  * `CVE-2024-2398`: HTTP/2 push headers memory-leak
+* `USN-6697-1` Bash vulnerability:
+  * `CVE-2022-3715`: A flaw was found in the bash package, where a heap-buffer overflow can occur in valid parameter_transform. This issue may lead to memory problems.
+
+
+## Platform Changes
+---
+
+This section is for the platform operators at `cloud.gov` to highlight changes to Cloud Foundry components, this is likely not of interest for developers using the platform.
+
+### cf-networking 3.44.0 to 3.45.0
+
+* Go dependency bumps
+* Added config for staticcheck
+* Removed references to consul from documentation
+
+
+### cf-smoke-tests 42.0.137 to 42.0.140
+
+* Bump golang, cf-cli and smoke_tests packages
+
+
+### diego 2.95.0 to 2.97.0
+
+* Bump to golang 1.21.8
+* Golang package dependency bumps
+  * Includes moving from docker v20 to docker v26
+  * Includes moving from pgx v3 to pgx v5
+* Remove references to Consul from boshrelease + documentation
+* Many fixes to remove deprecated code and otherwise fix linter violations.
+* Adds support for comma-delimited destinations in ASGs
+  * See: [cloudfoundry/executor#96](https://github.com/cloudfoundry/executor/pull/96) and [cloudfoundry/bbs#94](https://github.com/cloudfoundry/bbs/pull/94)
+* Bump bosh package blobs
+  * Bumps `jq` to `1.7.1`
+  * Bumps `tar`
+* Bump go.mod dependencies
+
+
+### garden-runc 1.50.0 to 1.51.0
+
+* Adds `grootfs.routine_gc` property, which allows operators to configure garden to grootfs to clean up unused container image layers whenever new containers are created.
+  * Previously, to achieve this, operators had to set `grootfs.reserved_space_for_other_jobs_in_mb` to the same value as the ephemeral disk, which is not always easy to obtain programatically.
+* Bump go dependencies
+
+
+### routing 0.293.0 to 0.294.0
+
+* Bump haproxy to 2.8.7
+* update templates to not refer to consul
+* Default the routing_api.enabled_api_endpoints to mtls
+* Update template test for mTLS routing api default
+* Upgrade cf-cli-8-linux
+
+
+### silk 3.44.0 to silk 3.45.0
+
+* Fixes to stop using deprecated code in tests (net.Error.Temporary())
+* Remove consul references from documentation
+* Fix issues bumping the code.cloudfoundry.org/executor package
+* Go package bumps
+
+
+### uaa 77.2.0 to 77.4.0
+
+* Bump to UAA v77.4.0
+* Misc
+  * Fixes a regression introduced in commit d10922a where we stopped reading the signatureAlgorithm from the properties file and only used the SHA1 default.
+* Dependency Bumps
+  * Add jwtClientAuthentication in Oauth provider examples
+  * Bump github.com/cloudfoundry/bosh-utils from 0.0.452 to 0.0.453
+  * Bump minitest from 5.22.2 to 5.22.3
+  * Bump bigdecimal from 3.1.6 to 3.1.7
+  * Bump nokogiri from 1.16.2 to 1.16.3
+  * Bump github.com/onsi/gomega from 1.31.1 to 1.32.0 in /src/acceptance_tests
+
+
+
+## Final Note
+
+Enjoy the eclipse!   
```

Comment:
I like this, but maybe we should link out to NASA or NOAA so we're not confused looking back at this in 5 years?